packages/oui-color/README.md

```diff
@@ -395,3 +395,13 @@
     </span>
   </div>
 </div>
+
+
+## Text color
+
+Text can be colored thanks to helper classes. The name of the classes is `oui-color-text_oui-color-` followed by the name of the color.
+
+```html:preview
+<p class="oui-color-text_oui-color-atlantis">This is atlantis</p>
```

Comment:
`oui-color-text_oui-color-atlantis` should be `oui-color-atlantis`.

Reply:
Are you sure ? This won't convey the idea that it's text color

Reply:
In the same way as `oui-icon-*`. Shorter is better.